mappings/net/minecraft/world/gen/decorator/PlacementModifiers.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_6817 net/minecraft/world/gen/decorator/PlacementModifiers
```

Comment:
Should be called `PlacedFeatures`, as it contains the registration method for `PlacedFeature`s as well as the method for getting a default one for the defaulted registry.